Makefile

```diff
@@ -124,29 +124,43 @@ enwiki_tuning_reports: \
 
 ########################## Basque Wikipedia ####################################
 
+
+datasets/euwiki.sampled_revisions.172_balanced.json:
+	# from https://quarry.wmflabs.org/query/36633
+	wget https://quarry.wmflabs.org/run/379904/output/0/json-lines -qO- > $@
+
 datasets/euwiki.human_labeled.400.json:
 	./utility fetch_labels \
-                https://labels.wmflabs.org/campaigns/euwiki/79/ > $@
+	 https://labels.wmflabs.org/campaigns/euwiki/79/ > $@
+
+datasets/euwiki.human_labeled.v2.172_balanced.json:
+	./utility fetch_labels \
+	 https://labels.wmflabs.org/campaigns/euwiki/91/ > $@
 
 datasets/euwiki.human_labeled.300_balanced.json: \
 		datasets/euwiki.human_labeled.400.json
 	(cat $< | grep '"wp10": "Stub"' | shuf -n 50; \
 	 cat $< | grep '"wp10": "Start"' | shuf -n 50; \
- 	 cat $< | grep '"wp10": "C"' | shuf -n 50; \
- 	 cat $< | grep '"wp10": "B"' | shuf -n 50; \
- 	 cat $< | grep '"wp10": "GA"' | shuf -n 50; \
- 	 cat $< | grep '"wp10": "FA"' | shuf -n 50) > $@
+	 cat $< | grep '"wp10": "C"' | shuf -n 50; \
+	 cat $< | grep '"wp10": "B"' | shuf -n 50; \
+	 cat $< | grep '"wp10": "GA"' | shuf -n 50; \
+	 cat $< | grep '"wp10": "FA"' | shuf -n 50) > $@
+
+datasets/euwiki.human_labeled.472_balanced.json: \
+		datasets/euwiki.human_labeled.300_balanced.json \
```

Comment:
This remains as 300_balanced instead of 472_balanced?

Reply:
Right.  In this rule, we're combining the 300 with the 172 to get to 472.